.travis.yml

```diff
@@ -42,7 +44,7 @@ install: # Use this to install any prerequisites or dependencies necessary to ru
   # Install dependencies for source repos
   - rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO -y
 before_script: # Use this to prepare your build for testing e.g. copy database configurations, environment variables, etc.
+  - export PYTHONPATH=/usr/lib/python2.7/dist-packages:$PYTHONPATH
   - source /opt/ros/$ROS_DISTRO/setup.bash  
 script: # All commands must exit with code 0 on success. Anything else is considered failure.
-  - catkin_make -j2
-
+  - catkin_make -j2 --cmake-args -Wno-dev
```

Comment:
I'd prefer to have all warnings if they make sense.
On 9 Dec 2015 15:42, "Enrique Fernández Perdomo" <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/ros-controls/ros_control/pull/224#discussion_r47106490>
> :
>
> >    - source /opt/ros/$ROS_DISTRO/setup.bash
> >  script: # All commands must exit with code 0 on success. Anything else is considered failure.
> > -  - catkin_make -j2
> > -
> > +  - catkin_make -j2 --cmake-args -Wno-dev
>
> From man cmake:
>
>        -Wno-dev
>               Suppress developer warnings.
>
>               Suppress  warnings  that are meant for the author of the CMakeLists.txt
>               files.
>
> @otamachan <https://github.com/otamachan> Is this really needed? Which
> warnings do you see?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ros-controls/ros_control/pull/224/files#r47106490>.
>